packages/react/src/contexts/ClerkContextProvider.tsx

```diff
@@ -47,10 +47,17 @@ export function ClerkContextProvider(props: ClerkContextProvider): JSX.Element |
   const clerkCtx = React.useMemo(() => ({ value: clerk }), [clerkLoaded]);
   const clientCtx = React.useMemo(() => ({ value: state.client }), [state.client]);
 
-  const { sessionId, session, userId, user } = derivedState;
+  const { sessionId, session, userId, user, organization } = derivedState;
   const actor = session?.actor || null;
-
-  const authCtx = React.useMemo(() => ({ value: { sessionId, userId, actor } }), [sessionId, userId, actor]);
+  const orgId = organization?.id;
```

Comment:
If an org is not selected, shouldn't this be `null`?

Reply:
Oops. Yes. This mistake seems to be repeating itself. It's quite easy to miss!

Reply:
Actually, we do not really need to differentiate between `null` and `undefined` for `orgId` and `orgRole` inside the `ClerkContextProvider`, since `useAuth` will set them accordingly depending on the `session`.